wechatpay/src/main/java/com/adyen/checkout/wechatpay/internal/ui/DefaultWeChatDelegate.kt

```diff
@@ -110,8 +110,7 @@ internal class DefaultWeChatDelegate(
 
     @SuppressWarnings("ReturnCount")
     override fun handleAction(action: Action, activity: Activity) {
-        @Suppress("UNCHECKED_CAST")
-        val sdkAction = (action as? SdkAction<WeChatPaySdkData>)
```

Comment:
Only doing check would crash if you pass `SdkAction<TwintSdkData>`, so I changed it

Reply:
If you pass `SdkAction<TwintSdkData>`, doesn't `(action as? SdkAction<WeChatPaySdkData>)` return `null`?

Reply:
Nope, because of type erasure. `action` would be cast to `SdkAction<*>` and it doesn't care about the generic